__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const testData = require("../db/data/test-data");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("should return an array of topics, each with a 'slug' and 'description' property", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Array.isArray(body.topics)).toBe(true);
+        body.topics.forEach((topic) => {
+          expect(topic).toHaveProperty("slug");
+          expect(topic).toHaveProperty("description");
+        });
```

Comment:
Almost. We'd like you to check that the slug and description keys are there but we'd also like to check that the values are strings. 

We also need you to do a length check. Otherwise, our forEach could return a false positive. 

Thanks :)